contrib/puppet-lumogon/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "puppet-lumogon",
```

Comment:
should be puppetlabs-lumogon as the forge and github namespace is still puppetlabs alas